content-api/content-actors/src/main/scala/org/sunbird/content/util/CopyManager.scala

```diff
@@ -127,6 +127,11 @@ object CopyManager {
         val keysNotPresent = ContentConstants.REQUIRED_KEYS.filter(key => emptyCheckFilter(request.getRequest.getOrDefault(key, "")))
         if (keysNotPresent.nonEmpty)
             throw new ClientException(ContentConstants.ERR_INVALID_REQUEST, "Please provide valid value for " + keysNotPresent)
+        if (StringUtils.equalsIgnoreCase(request.getRequest.getOrDefault(ContentConstants.COPY_TYPE, ContentConstants.COPY_TYPE_DEEP).asInstanceOf[String], ContentConstants.COPY_TYPE_SHALLOW) &&
+            StringUtils.isNotBlank(request.get(ContentConstants.COPY_SCHEME).asInstanceOf[String]))
+            throw new ClientException(ContentConstants.ERR_INVALID_REQUEST, "Cannot Shallow copy content, with any copy scheme.")
```

Comment:
Message: "Content can not be shallow copied with copy scheme."

and above you added copyscheme in request.context. But here you are reading value from request.request.

Reply:
Will fix it.